server/test/christmas-tree-forest-controllers.spec.es6

```diff
@@ -6,7 +6,7 @@ const { createForest, destroyAll } = require('./data/db-helper.es6');
 
 const { expect } = chai;
 
-describe('christmas tree controller forest tests', () => {
+xdescribe('christmas tree controller forest tests', () => {
```

Comment:
Can we just delete this test, or is this being used in the pipeline?

Reply:
We will need to handle this in our christmas tree cleanup

Reply:
> We will need to handle this in our christmas tree cleanup

Ah I should have thought of that. Good call, thanks Mike.